shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -100,6 +100,70 @@ - (void)addInternalPlugins;
 - (void)deregisterNotifications;
 @end
 
+@interface KeyboardAnimationView : UIView
+@property(nonatomic) fml::TimePoint startTime;
+@property(nonatomic) NSTimeInterval duration;
+@property(nonatomic) CGFloat from;
+@property(nonatomic) CGFloat to;
+@property(nonatomic) double lastComplete;
+@end
```

Comment:
Is This `lastComplete ` used? Or we can delete it🙂.

Reply:
bye bye! ha